src/main/java/org/opensearch/performanceanalyzer/rca/store/OpenSearchAnalysisGraph.java

```diff
@@ -433,6 +443,28 @@ public void construct() {
                         shardRequestCacheClusterRca,
                         highHeapUsageClusterRca));
 
+        // Search Back Pressure Service RCA enabled
+        SearchBackPressureRCA searchBackPressureRCA =
+                new SearchBackPressureRCA(RCA_PERIOD, heapMax, heapUsed, searchbp_Stats);
+        searchBackPressureRCA.addTag(
+                RcaConsts.RcaTagConstants.TAG_LOCUS,
+                RcaConsts.RcaTagConstants.LOCUS_DATA_CLUSTER_MANAGER_NODE);
+        searchBackPressureRCA.addAllUpstreams(Arrays.asList(heapMax, heapUsed, searchbp_Stats));
+
+        // Search Back Pressure Service Cluster RCA enabled
+        SearchBackPressureClusterRCA searchBackPressureClusterRCA =
+                new SearchBackPressureClusterRCA(RCA_PERIOD, searchBackPressureRCA);
+        searchBackPressureClusterRCA.addTag(
+                RcaConsts.RcaTagConstants.TAG_LOCUS,
+                RcaConsts.RcaTagConstants.LOCUS_CLUSTER_MANAGER_NODE);
+        searchBackPressureClusterRCA.addAllUpstreams(
+                Collections.singletonList(searchBackPressureRCA));
+        searchBackPressureClusterRCA.addTag(
+                RcaConsts.RcaTagConstants.TAG_AGGREGATE_UPSTREAM,
+                RcaConsts.RcaTagConstants.LOCUS_DATA_NODE);
+
+        // To Do SearchBackPressure Decider
```

Comment:
nit: use `TODO` instead of `To Do` for get better IDE support

Reply:
Fixed